### DISCUSSION TITLE
7 segment display

### DISCUSSION THREAD

**Discussion by DHMike57** [DigitalLogicSimCommunity/Digital-Logic-Sim-CE#75] [General]
I've just submitted a PR #74 porting over the 7-seg display from the original.  Unlike the original I couldn't see how to get the nifty highlighting of the segments on mousing the pins.  The pin layout is the same, though.  I  also used fixed sizing as in the display chip, which has the effect of disabling scaling.  Maybe someone who has actually seen Unity before (unlike me) could enhance it, but for now it seems to work fine.  

I also disabled the name (like the original) since it is obvious what the chip is from its appearance.

Binaries are available at [this page](https://github.com/ml57/Digital-Logic-Sim-CE/actions/runs/4418095425) if anyone wants to test (I believe you have to be logged in to github to see them).  The 7-seg is in the misc category and you need to scroll to see it.

Thanks to the authors for this fork - it has some nice QoL features!

Mike

**Comment by DHMike57** [comment]
Fixed the scaling issue, moved to `comp` folder and pins don't show red as in the original.  Binaries for testing [here](https://github.com/ml57/Digital-Logic-Sim-CE/actions/runs/4457407361), with changes in the `sevenseg` branch.

Mike

**Reply by phillipfu96** [reply]
404 not found 😢 , how can i get the binaries??

**Comment by DHMike57** [comment]
Github deleted the build files, either due to inactivity or changes in the api. I triggered a rebuild and you can get them [here](https://github.com/DHMike57/Digital-Logic-Sim-CE/actions/runs/11355940756) provided you are logged in to github.  Note that they are a few changes out of date now.

**Reply by phillipfu96** [reply]
Thank you sooo much!
